docs/releasehistory.rst

```diff
@@ -12,7 +12,8 @@ Changed behaviors
 Bugfixes
 --------
 - Bug in returning velocities when propagating replicas. Fixed by using ``ignore_velocities=False`` in ``_propagate_replica``. Issue `#531 <https://github.com/choderalab/openmmtools/issues/531>`_ (PR `#602 <https://github.com/choderalab/openmmtools/pull/602>`_).
-
+- Bug in equilibration detection when user specifies either ``statistical_inefficiency`` or ``n_equilibration_iterations`` (but not both). Fixed by preventing user from specifying only the former, and updating the behavior for the latter to include ``t0`` in the returned ``n_equilibration_iterations``. Issue `#609 <https://github.com/choderalab/openmmtools/issues/609>`_ (PR `#610 <https://github.com/choderalab/openmmtools/pull/610>`_). 
```

Comment:
Instead of just "Bug in X", can we say what happened in that case, or just state that the behavior was undefined? For example, "If the user specified `statistical_inefficiency` but not `n_equilibration_detection`, this would cause problem X. This is now fixed by ..." These kinds of statements get us in the habit of clearly communicating with users (and ourselves!) what went wrong to help evaluate if folks need to re-run calculations.

Reply:
Done